vignettes/cookbook.Rmd

```diff
@@ -11,10 +11,11 @@ vignette: >
 knitr::opts_chunk$set(
   collapse = TRUE,
   comment = "#>",
-  fig.width = 7,
+  fig.width = 670/72,
```

Comment:
@matthewstern I'm keeping the width at 670/72 for the "finalize" and "cookbook" vignettes to enforce correct layouts. Smaller widths can cause havoc with long titles/captions, especially in cookbook.